rpm-deploy.sh

```diff
@@ -53,12 +53,10 @@ BASEDIR=$(dirname "$SCRIPT_PATH")
 
 if rpm -q "salt"  > /dev/null 2>&1;
 then
-    # Release/Prod environment
-    ldap_admin_pwd=$(salt-call pillar.get openldap:iam_admin:secret --output=newline_values_only)
-    ldap_admin_pwd=$(salt-call lyveutil.decrypt openldap "${ldap_admin_pwd}" --output=newline_values_only)
+  ldap_admin_pwd=$(s3cipher --use_base64 --key_len  12  --const_key  openldap 2>/dev/null)
```

Comment:
remove this password creation from this script, and along with this, do changes in authserver to not read password from authserver.properties.

Reply:
We will handle this in next PR.